### PR TITLE
created search branch and input search component

### PR DESCRIPTION
**Pull request by frugalcodes**

### CODE REVIEW

File: bookfinder/tsconfig.json [hunk 1]
```diff
@@ -23,6 +23,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "src/app/components/SearchInput.tsx"],
```
Comment (ceifeirocv) [inline bookfinder/tsconfig.json]: Why did you add this line

Comment (frugalcodes) [reply bookfinder/tsconfig.json]: i dont think it was added intentionally, because i had nothing to do with the tsconfig.json file. probably it was added when i ran "npm run build"

Comment (EmmanuelOloke) [reply bookfinder/tsconfig.json]: I don't think you need to run `npm run build` before pushing your code. You can remove this since `.tsx` is already captured in the flags.

File: bookfinder/src/app/components/SearchInput.tsx
Comment (EmmanuelOloke) [inline bookfinder/src/app/components/SearchInput.tsx:1]: I don't know if I'm missing anything, but this is what I can see after fetching the `search` branch.

Screenshot 2023-08-21 at 11 32 36

This is what the design suggests here:

Screenshot 2023-08-21 at 11 31 49

Did you make a reference to the design?

https://www.figma.com/file/ep8G9FmBjyI97BPPUsV9hZ/book-price-finder?type=design&node-id=0-1&mode=design&t=3fMKMWlcvAITkWbU-0

File: bookfinder/tsconfig.json
Comment (EmmanuelOloke) [inline bookfinder/tsconfig.json:1]: I think you only need to remove this `"src/app/components/SearchInput.tsx"` from the line. The other flags are still needed.

Comment (EmmanuelOloke) [reply bookfinder/tsconfig.json:1]: I think line 26 should not be removed entirely, only the highlighted parts as suggested in the last comment.

**Review by ceifeirocv — DISMISSED** [review]

**Review by EmmanuelOloke — CHANGES_REQUESTED** [review]

**Review by EmmanuelOloke — APPROVED** [review]
Replace line 26 in `tsconfig.json`

**Review by yildiztugba — APPROVED** [review]

**Review by Pio-js — APPROVED** [review]
